source/guides/deployment.rst

```diff
@@ -219,12 +219,14 @@ Changelogs
     Mattermost Cloud </install/cloud-changelog>
     Mobile Apps </deploy/mobile-app-changelog>
     Desktop Apps </install/desktop-app-changelog>
+    Deprecated Features </install/deprecated-features>
```

Comment:
I'm wondering if the "Upgrading Mattermost" section might be better.

Reply:
- The benefit of having it here is that it's in the same vein as changes in general. I'd be more inclined to have it here. 
- The benefit of having it in the Upgrade section is that it's important information that could relate to changes after upgrade. I'd say the link is a bit more tenuous but there is still a link.

I'm more inclined for option 1. @cwarnermm, what are your thoughts?

Reply:
Tough call! You've both done a great job of identifying the two most natural places to put this page. Are customers more likely to look for deprecation details around the changelogs or the upgrade details?

I'm inclined to keep this page with the changelogs because it answers the question "what's changed?".